region/norfair/east/Lava Farm Tunnel.json

```diff
@@ -1192,6 +1233,40 @@
         "The pixel window is larger and higher with Morph and an X-Ray stand-up, in which case a standing jump is used."
       ]
     },
+    {
+      "link": [4, 3],
+      "name": "R-Mode Forced Standup Door Lock Skip",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        "canRModeStandupClip",
+        "canInsaneJump",
+        {"or": [
+          "canMidAirMorph",
```

Comment:
I felt a difference with springball but not with jump-morph.
Pause abuse + visual cue was as good as anything other than spring ball for me.